app/models/game.rb

```diff
@@ -0,0 +1,4 @@
+class Game < ApplicationRecord
+  has_many	:pieces
+  has_many	:users
```

Comment:
It will work fine.. But actually a game has only 2 users.. A good way would be to use

`  belongs_to :white_player, :class_name => 'User', :foreign_key => 'user_id'
 belongs_to :black_player, :class_name => 'User', :foreign_key => 'user_id'
`

[Here](https://stackoverflow.com/questions/34697634/same-model-for-two-belongs-to-associations-migration) is a reference on how to do it:

Reply:
Looks like it's the way forward, but the way to achieve it and how it works is not crystal clear to me... just made an attempt on a branch I pushed on github, hope I'm not too far off...